irobot_edu_sdk/create3.py

```diff
@@ -2,6 +2,8 @@
 # Licensed under 3-Clause BSD license available in the License file. Copyright (c) 2020-2024 iRobot Corporation. All rights reserved.
 #
 
+#-------------See changes in lines 164 through 185 to address premature timeout when navigating to origin using navigate_to()-----------------
+
```

Comment:
Please remove these lines. These, and the commentary below are helpful, but they belong in the pull request, not in the final source.